app.ts

```diff
@@ -73,6 +73,9 @@ import type {Language, LanguageKey} from './types/languages.interfaces.js';
 // Used by assert.ts
 global.ce_base_directory = new URL('.', import.meta.url);
 
+// Used by d8.ts
+global.handlerConfig = null;
```

Comment:
Super minor but it would be good to add a typescript type here. Can be done in another pr/commit though, I don't want to hold this up.

Reply:
Thank you! I've uploaded something for this in https://github.com/compiler-explorer/compiler-explorer/pull/5858